.ci/Jenkinsfile

```
@@ -188,7 +188,11 @@ def generateStep(version){
 }
 
 def releasePackages(){
-  withSecretVault(secret: 'secret/apm-team/ci/apm-agent-python-twine',
+  def vault_secret = 'secret/apm-team/ci/apm-agent-python-pypi-test'
```

More groovysh:
```suggestion
  def vault_secret = (env.REPO_URL == 'https://upload.pypi.org/legacy/') ? 'secret/apm-team/ci/apm-agent-python-pypi-prod' : 'secret/apm-team/ci/apm-agent-python-pypi-test'
```

The above suggestion replaces from 191...194